spec/client/ics-028-wasm-client/README.md

```diff
@@ -0,0 +1,448 @@
+---
+ics: 28
+title: Wasm Client
+stage: draft
+category: IBC/TAO
+kind: instantiation
+implements: 2
+author: Parth Desai <parth@chorus.one>
+created: 2020-10-13
+modified: 2020-10-13
+---
+
+## Synopsis
+
+This specification document describes an interface to a client (verification algorithm) stored as a Wasm bytecode for a blockchain.
+
+### Motivation
+
+Wasm based light clients are decoupled from SDK source code, which enables one to upgrade existing light client or add support for other blockchain without modifying SDK source code.
+
+### Definitions
+
+Functions & terms are as defined in [ICS 2](../ics-002-client-semantics).
+
+`currentTimestamp` is as defined in [ICS 24](../ics-024-host-requirements).
+
+`Wasm Client Code` refers to Wasm bytecode stored in the client store, which provides a target blockchain specific implementation of [ICS 2](../ics-002-client-semantics).
```

Comment:
Is the wasm client code stored in the clientID-prefixed store provided by IBC or it it stored in a KV store owned by WASM?

Reply:
the Wasm client code is stored in the KV store owned by 28-wasm module.

Example entry:
```
ibc/code_id/d6537444f6df8686bdfefc3dc42441cd1d332ac608d27ef5903dd083a537e85f
```

Source: https://github.com/ChorusOne/ibc-go/blob/ics_28_wasm_client/modules/core/28-wasm/keeper/keeper.go#L61